pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -79,6 +79,24 @@ type CiliumBGPNeighbor struct {
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:validation:Maximum=4294967295
 	PeerASN int `json:"peerASN"`
+	// ConnectRetryTime defines the initial value for the BGP ConnectRetryTimer (RFC 4271, Section 8).
+	// The default value for the ConnectRetryTime (if empty or zero) is 120 seconds.
+	//
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Format=duration
+	ConnectRetryTime metav1.Duration `json:"connectRetryTime,omitempty"`
```

Comment:
Let's add a unit here as well.

Reply:
Added a unit test for validation ConnectRetryTime - but not sure if you meant that? :)